cfpb/consumerfinance.gov
# Link to static html list from no-js raf tool

Alternative to https://github.com/cfpb/consumerfinance.gov/pull/6994

Instead of shipping a bunch of html to every user, think just gives `no-js` users a *link* to said html. Here's a link to that html: https://files.consumerfinance.gov/data/rental-assistance.html

I think this is a worse experience for `no-js` users than https://github.com/cfpb/consumerfinance.gov/pull/6994, but definitely a better experience for the 99% other users, so probably better overall.

Testing:
 - Pull branch
 - Visit http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/
 - Things should work as normal
 - Open devtools and use cmd + shift +p to open the run panel, type Disable Javascript and run that command when it populates
 - Refresh the page
 - The page should render with the following notification:
 
<img width="902" alt="Screen Shot 2022-04-05 at 4 37 54 PM" src="https://user-images.githubusercontent.com/1558033/161844955-c6495f00-a01c-4075-aa17-e8d276be5670.png">

That link should point to https://files.consumerfinance.gov/data/rental-assistance.html
